test-case-reporting/test/e2e.test.ts

```diff
@@ -49,9 +49,11 @@ describe.skip('end-to-end', () => {
     await truncateAll(db);
   });
 
-  describe('when one post request is received', async () => {
+  describe('when one post request is received', () => {
     it('updates the database if the post request is good', async () => {
-      await request(app)
+      let res;
+
+      res = await request(app)
```

Comment:
Done.

Reply:
Doesn't appear so yet

Reply:
It looks like it to me -- 
![image](https://user-images.githubusercontent.com/12011689/88568212-98ca5700-d006-11ea-9bcc-8a50f3198aa5.png)

Reply:
By "the above line" I didn't mean the blank one :laughing: 
`let res = await request(app)`

Reply:
Ohh, okay. That's kind of a funny misunderstanding -- I was particularly confused because the preview showed the blank line as unchanged -- 
![image](https://user-images.githubusercontent.com/12011689/88574061-b3ed9480-d00f-11ea-9597-3031d039a225.png)

Done.